pre_commit_hooks/check_case_conflict.py

```diff
@@ -31,8 +43,7 @@ def find_conflicting_filenames(filenames: Sequence[str]) -> int:
 
     if conflicts:
         conflicting_files = [
-            x for x in repo_files | relevant_files
-            if x.lower() in conflicts
+            x for x in repo_files | relevant_files if x.lower() in conflicts
```

Comment:
no idea why this line changed, please leave the original

Reply:
it is formatted by the `pre-commit`

Reply:
will revert